website/src/get-started/wordcount-example.md

```diff
@@ -1362,7 +1379,18 @@ PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.Cou
 ```
 
 ```py
-# This feature is not yet available in the Beam SDK for Python.
+class CountWordsFn(Dofn):
```

Comment:
@rakeshcusat have you tested running snipet? I am not sure it will work, because:
- `Dofn` is misspelled. Should be `DoFn`
- `process` method does not return or yield.
- `FlatMap` expects a lambda not a DoFn.

Reply:
@aaltay thanks for looking into this. I definitely not ran this snippet. I will fix these issues and make sure it is executable. 

I have another doubt since we have removed the metrics statement so nothing in the code is tracking the number of words. is leaving a comment good enough?

Reply:
So this is about reusing transforms, right and java version is the single line `PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.CountWords());`

We can probably reduce this too

`word_counts = windowed_words | CountWords()`

And if you would like to add code for what is CountWords, perhaps you can link to the pre-existing example transform (https://github.com/apache/beam/blob/e65c176a9f34e45d408281e1101a2ae54cef0f6c/sdks/python/apache_beam/examples/wordcount_debugging.py#L91)

What do you think?

Reply:
For consistency shake I will use the single line instead. 
`word_counts = windowed_words | CountWords()`

Reply:
Anyway, full python code example link is already provided down below. So we don't need to provide the repeated information here.